09.mesh-modeler/02.item-editor/docs.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Item Editor'
+taxonomy:
+    category:
+        - docs
+visible: true
+---
+
+[TOC]
+
+The Item Editor allows you to edit items (.Item.Gbx files) that can be placed in maps. It can be accessed directly from the *Title tools* menu, but it is recommended to open it from the map editor, via the *Item Edition* panel, at the left bottom corner of the screen in Item mode.
```

Comment:
```suggestion
The Item Editor allows you to edit items (`.Item.Gbx` files) that can be placed in maps. It can be accessed directly from the *Title tools* menu, but it is recommended to open it from the map editor, via the *Item Edition* panel, at the left bottom corner of the screen in Item mode.
```